tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -228,6 +228,22 @@
     "job.result().get_unitary(qc, decimals=3)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Note on U gate deprecation\n",
```

Comment:
I added in the alert-info block, however it doesn't format on reviewNB. Cloned/compared to how it was done in the fall-2021 challenge and looks right. @HuangJunye if you can confirm this is addressed since not sure if there is a way to validate.

Reply:
I went ahead and fix the info box. I am not sure why ReviewNB is still not rendering the box, but i can verify that local build is rendering correctly. 
<img width="864" alt="image" src="https://user-images.githubusercontent.com/7631333/173538139-294e5acd-48b2-46b3-8163-e190bf454a81.png">

I also added bullet points and use markdown code block (see below, or this article for more details: https://markdown.land/markdown-code-block) 

```markdown

```python


```

Reply:
This PR https://github.com/Qiskit/qiskit-tutorials/pull/1332 will add documentation about how to do local build so that you can see exactly how it will look on the website.